.vib/node/cypress/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Why do we need functional tests for a runtime as Node?

Reply:
Are we delivering a demo app or similar?

Reply:
Yes, the chart itself has params to clone an app from a repo and deploy it 😁 

https://github.com/bitnami/charts/blob/6d6c64ff11b4953b89f73218bc4b907345247cb1/bitnami/node/values.yaml#L446-L452